lib/service/Service.ts

```diff
@@ -330,6 +330,34 @@ class Service extends EventEmitter {
     return this.orderBook.removePair(pairId);
   }
 
+  public executeSwap = async (args: { orderId: string, pairId: string, peerPubKey: string, quantity: number }): Promise<SwapResult> => {
```

Comment:
Another minor thing, I believe this file also tries to follow alphabetical order for methods to align with the proto file and GrpcService, so could you move this method up a bit as well?